src/components/TimecodeTracker/TimecodeTracker.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import {
+  getTimecodeGameLength,
+  getTimecodeTotalLength,
+} from '../../utils/Timecode'
+import TimecodeInput from './TimecodeInput'
+import TimecodeResult from './TimecodeResult'
+
+export default function TimecodeTracker() {
+  const [timecodes, setTimecodes] = useState(['', '', ''])
+
+  return (
+    <>
+      <form onKeyDown={handleEnter}>
+        {timecodes.map((_, index) => (
+          <TimecodeInput
+            title={index < 2 ? 'Halbzeit ' + (index + 1) : 'O-Töne'}
+            index={index}
+            key={index}
+            onChange={handleChange}
+          />
+        ))}
+      </form>
+      <TimecodeResult
+        text="Länge Spiel"
+        result={getTimecodeGameLength(timecodes)}
+      />
+      <TimecodeResult
+        text="Länge gesamt"
+        result={getTimecodeTotalLength(timecodes)}
+      />
+    </>
+  )
+
+  function handleChange(event, index) {
```

Comment:
Adding function parameters as object?

Reply:
what do you mean exactly?

Reply:
function handleChange({event, index})

Reply:
hmm i think, this would make the code more complex in writing. i need to change the part where the function is invoked too.
Why would you do it? would the code be better?